config/initializers/devise.rb

```diff
@@ -190,7 +190,7 @@
   # ==> Configuration for :timeoutable
   # The time you want to timeout the user session without activity. After this
   # time the user will be asked for credentials again. Default is 30 minutes.
-  # config.timeout_in = 30.minutes
+  config.timeout_in = 2.hours
```

Comment:
I don't think we should change this.

It's only a small number of users who don't get their code in 10 minutes, and the Devise timeout doesn't affect the login journey, so the only place in which this change is relevant is if the user is already logged in when we prompt them to go through 2fa (eg, when they're changing their phone number).  Given hardly any users are affected, I think we can probably just live with some of them having to resend the code occasionally.